pymongo/server.py

```diff
@@ -159,7 +186,10 @@ def run_operation(self, sock_info, operation, read_preference, listeners, unpack
             elif operation.name == "explain":
                 res = docs[0] if docs else {}
             else:
-                res = {"cursor": {"id": reply.cursor_id, "ns": operation.namespace()}, "ok": 1}
+                res = {
+                    "cursor": {"id": reply.cursor_id, "ns": operation.namespace()},
+                    "ok": 1,
+                }
```

Comment:
Was this change needed or can it be undone? (we prefer not to reformat unnecessarily to avoid code churn).

Reply:
I can undo it :)